runtime/oti/j9nonbuilder.h

```diff
@@ -3020,6 +3020,7 @@ typedef struct J9Class {
 	struct J9Class* nestHost;
 #endif /* defined(J9VM_OPT_VALHALLA_NESTMATES) */
 	struct J9FlattenedClassCache* flattenedClassCache;
+	UDATA hotFieldOffset;
```

Comment:
The same field needs to be added to `J9ArrayClass` as well.  The two structs must be identical as the Array version provides an overlay with different names for some of the fields

Reply:
👍